packages/helpers/package.json

```diff
@@ -41,6 +41,8 @@
     "@ckb-lumos/bi": "^0.19.0-alpha.3",
     "@ckb-lumos/config-manager": "^0.19.0-alpha.3",
     "@ckb-lumos/toolkit": "^0.19.0-alpha.3",
+    "@types/blake2b": "^2.1.0",
+    "blake2b": "^2.1.3",
```

Comment:
I think we CAN'T
because in https://github.com/ckb-js/lumos/blob/e33aaa10d831edd58e904cb2215ea3148c37fba3/packages/base/src/utils.js#L11
the CKBHasher defined as
```
 this.hasher = blake2b(32, null, null, bytifyRawString("ckb-default-hash"));
```
we need change blake2b in differient hash algorithms.